src/host/host.ts

```diff
@@ -58,8 +61,6 @@ export function initialize(dtWindow: IDevToolsWindow): void {
         // Override the resource loading once the window has loaded so that we can control it
         const resourceLoader = ToolsResourceLoader.overrideResourceLoading(dtWindow.Root.Runtime);
         dtWindow.InspectorFrontendHost.setResourceLoader(resourceLoader);
-
-        dtWindow.importScriptPathPrefix = dtWindow.importScriptPathPrefix.replace('null', 'vscode-webview-resource:');
```

Comment:
@vidorteg  - we no longer need this, right?

Reply:
IIUC, this should be safe to remove, as it was only needed to transform loading requests from inside the iframe into a vscode webview resource, but that should all be done automatically now since we are in the same webview/domain as the resource loader